src/Microsoft.ML.Data/Transforms/LabelConvertTransform.cs

```diff
@@ -223,7 +224,8 @@ public override ValueGetter<VBuffer<TValue>> GetGetter<TValue>()
             {
                 ValueGetter<VBuffer<TValue>> getter = _getter as ValueGetter<VBuffer<TValue>>;
                 if (getter == null)
-                    throw Ch.Except("Invalid TValue: '{0}'", typeof(TValue));
+                    throw Ch.Except($"Invalid TValue: '{typeof(TValue)}', " +
+                            $"expected type: '{_getter.GetType().GetGenericArguments().First()}'.");
```

Comment:
Nit: shouldn't this be `_getter.GetType().GetGenericArguments().First().GetGenericArguments().First()` just as you did in Transposer.cs since this is a VBuffer?

Reply:
Thanks, this is bug

---
In reply to: [433884538](https://github.com/dotnet/machinelearning/pull/5189#discussion_r433884538) [](ancestors = 433884538)